src/features/metrics/components/MetricsSidebar.tsx

```diff
@@ -142,6 +143,7 @@ export function MetricsSidebar({
         </ScrollArea>
 
         {footer}
+        <SubmitBallotButton />
```

Comment:
MetricsSidebar is also rendered by MetricDetailsSidebar, that is used in `/pages/[domain]/metrics/[metricId]/index.tsx`

is a page that is not related to the ballot,  only shows information about an specific metric and shouldn't have the SubmitBallotButton.

The Button should be in BallotSidebar for now that is rendered only in the Ballot/Metrics pages